examples/libusb/boo.cpp

```diff
@@ -1,4 +1,20 @@
 #include <libusb.h>
 
 int main() {
+    libusb_device **devs;
```

Comment:
`PACKAGE_INTERNAL_DEPS_ID` of the package should be updated, otherwise, nothing will be tested, and everything will be taken from the cache:
- https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html

Reply:
As you can see there is no build triggered here:
- https://travis-ci.org/hjmallon/hunter/jobs/523397199

```
-- [hunter *** DEBUG *** 2019-04-23T09:35:19] Downloading DONE metafile (try #0 of 10):
-- [hunter *** DEBUG *** 2019-04-23T09:35:19]   https://raw.githubusercontent.com/ingenue/hunter-cache/master/239e635/libusb/1.0.22/cdb45a2/7b367dc/a23c16c/e5fa44f/da39a3e/CACHE.DONE
-- [hunter *** DEBUG *** 2019-04-23T09:35:19]   -> /home/travis/build/hjmallon/hunter/_testing/Hunter/_Base/Cache/meta/239e635/libusb/1.0.22/cdb45a2/7b367dc/a23c16c/e5fa44f/da39a3e/CACHE.DONE
-- [hunter *** DEBUG *** 2019-04-23T09:35:19] Downloading metafile (try #0 of 2):
-- [hunter *** DEBUG *** 2019-04-23T09:35:19]   https://raw.githubusercontent.com/ingenue/hunter-cache/master/239e635/libusb/1.0.22/cdb45a2/7b367dc/a23c16c/e5fa44f/da39a3e/cache.sha1
-- [hunter *** DEBUG *** 2019-04-23T09:35:19]   -> /home/travis/build/hjmallon/hunter/_testing/Hunter/_Base/Cache/meta/239e635/libusb/1.0.22/cdb45a2/7b367dc/a23c16c/e5fa44f/da39a3e/cache.sha1
-- [hunter] Cache HIT: libusb # <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
-- [hunter] Cache info: /home/travis/build/hjmallon/hunter/_testing/Hunter/_Base/Cache/meta/239e635/libusb/1.0.22/cdb45a2/7b367dc/a23c16c/e5fa44f/da39a3e/cache.sha1
-- [hunter *** DEBUG *** 2019-04-23T09:35:19] Locking directory: /home/travis/build/hjmallon/hunter/_testing/Hunter/_Base/Cache
```

Reply:
We will need to check those too:
- https://github.com/ruslo/hunter/blob/v0.23.158/cmake/modules/hunter_pkgconfig_export_target.cmake#L15-L18

Please update their `PACKAGE_INTERNAL_DEPS_ID`.